### PR TITLE
 docs: Continue to reorganize docs

### PR DESCRIPTION
**Pull request by murrant**
Make landing pages for /Installation, /API, etc.
Split API out into multiple docs
Properly use multi-level navigation
Add lower nav buttons (tops are bug so disable for now)
Add logo for fanciness
Refine index page a bit

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: doc/API/API-Docs.md [hunk 1]
```diff
@@ -1 +0,0 @@
-<meta http-equiv="refresh" content="0; url=/API" />
```
Comment (laf) [inline doc/API/API-Docs.md]: This is to preserve old links, why remove it?

File: doc/Support/FAQ.md [hunk 2]
```diff
@@ -268,7 +262,7 @@ There is a known issue for net-snmp, which causes it to report incorrect disk si
 
 #### <a name="faq8"> How do I add support for a new OS?</a>
 
-Please see [Supporting a new OS](../Developing/Support-New-OS.md)
+Please see [Supporting a new OS](/#Support-for-a-New-OS)
```
Comment (laf) [inline doc/Support/FAQ.md]: Links should stay pointing to markdown files, they should get converted correctly whilst ensuring if we change the theme again and a # isn't needed then it doesn't break.

File: doc/Support/Support-New-OS.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
 source: Support/Support-New-OS.md
-<meta http-equiv="refresh" content="0; url=/Developing/Support-New-OS/" />
+<meta http-equiv="refresh" content="0; url=/Developing/Support-New-OS/__" />
```
Comment (laf) [inline doc/Support/Support-New-OS.md]: What's the reason for the __?

File: doc/API/API-Docs.md (hunk 1 above)
Comment (murrant) [reply doc/API/API-Docs.md]: I thought it was unused in the docs, I'll look through the original mkdocs.yml again.

Comment (laf) [reply doc/API/API-Docs.md]: It's from the v1 theme. The links are in older installs of LibreNMS and most probably linked to from reddit and other places in old posts.

Comment (murrant) [reply doc/API/API-Docs.md]: This wasn't defined in mkdocs.yml, so it wasn't being built.  That is why I removed it.

File: doc/Support/Support-New-OS.md (hunk 3 above)
Comment (murrant) [reply doc/Support/Support-New-OS.md]: probably attempting to type in another window and I missed it.  Thanks.

File: doc/API/API-Docs.md (hunk 1 above)
Comment (murrant) [reply doc/API/API-Docs.md]: If you can tell me the URL it was supposed to be, I can fix it.

Comment (laf) [reply doc/API/API-Docs.md]: It was pre-mkdocs by the looks of it:
https://github.com/librenms/librenms/issues/7210

Shows that the old links were still in place.

Comment (laf) [reply doc/API/API-Docs.md]: You've fixed it anyway.

**Pull request merged**